tools/paquet/paquet.gemspec

```diff
@@ -0,0 +1,28 @@
+# coding: utf-8
+lib = File.expand_path('../lib', __FILE__)
+$LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
+require 'paquet/version'
+
+Gem::Specification.new do |spec|
+  spec.name          = "paquet"
+  spec.version       = Paquet::VERSION
+  spec.authors       = ["Elastic"]
+  spec.email         = ["info@elastic.co"]
+  spec.license       = "Apache License (2.0)"
+
+  spec.summary       = %q{Rake helpers to create a uber gem}
+  spec.description   = %q{This gem add a few rake tasks to create a uber gems that will be shipped as a zip}
+  spec.homepage      = "https://github.com/elastic/paquet"
+
+
+  spec.files         = `git ls-files -z`.split("\x0").reject do |f|
```

Comment:
since the number of files and directories here is quite small and using exec'ing out is also usually bad, I suggest manually listing the files here.